platform/mcu/csky/csky.mk

```diff
@@ -37,6 +37,9 @@ GLOBAL_CFLAGS += $(INCLUDEDIRS)
 
```

Comment:
pass the crypto testcases?

Reply:
Certainly,,, but only pass supported algorithms and mode.